keypoints/rse-cmdline-git.md

```diff
@@ -0,0 +1,33 @@
+- Use git config with the `--global` option to configure a user name,
+  email address, editor, and other preferences once per machine.
+
+- `git init` initializes a repository.
+- Git stores all of its repository data in the `.git` directory.
+
+- `git status` shows the status of a repository.
+- Files can be stored in a project’s working directory (which users see),
+  the staging area (where the next commit is being built up)
+  and the local repository (where commits are permanently recorded).
+- `git add` puts files in the staging area.
+- `git commit` saves the staged content as a new commit in the local repository.
+- Write a commit message that accurately describes your changes.
+
+- `git diff` displays differences between commits.
+- `git checkout` recovers old versions of files.
+
+- `git branch` creates a new branch where new features can be developed while leaving the master branch untouched.
```

Comment:
Again, more general: Git "branches" allow you to isolate a series of changes from each other until they're ready to be combined.